### DISCUSSION TITLE
Hide track after scrolling ends

### DISCUSSION THREAD

**Discussion by Darcrandex** [idiotWu/smooth-scrollbar#535] [Q&A]
By default, the track is hidden, only shown when scrolling, and hidden again when stationary. How should I configure?
look this [demo](https://grsmto.github.io/simplebar/examples/)

**Comment by sadeghbarati** [comment]
Hi,

https://github.com/idiotWu/smooth-scrollbar/tree/develop/docs#available-options-for-scrollbar

did you try the `alwaysShowTracks` option?

**Reply by Darcrandex** [reply]
yes, I set alwaysShowTracks as false, but doesn't work.

**Comment by sadeghbarati** [comment]
@Darcrandex 

Can you share a demo? I don't really understand what are you intended to do

`smooth-scrollbar` already does the same functionality

1. Scrollbar track is hidden by default
2. User scrolled -> scrollbar track become visible for secs
3. Scrollbar track become hidden again

**Comment by Darcrandex** [comment]
Thank you very much.
Here's the online [demo](https://stackblitz.com/edit/vitejs-vite-vhkbew?file=src%2FApp.jsx)
you can debug in `App.jsx[line:20]`
